src/modules/email-verification/email-verification.service.ts

```diff
@@ -71,4 +71,33 @@ export class EmailVerificationService {
 
     return activatedUser;
   }
+
+  async sendForgotPasswordEmail(to: string): Promise<boolean> {
+    try {
+      const resetToken = await this.generateResetPasswordToken(to);
+      const mailOptions = {
+        from: process.env.SMTP_USER,
+        to,
+        subject: 'Reset Password',
```

Comment:
It is not recommended to leave strings in the code. It is better to create new file called email.constants.ts. Here you create EMAIL_SUBJECT object which will have RESET_PASSWORD field = 'reset password'. 
And you should apply it to all the strings in the code preferably. Leaving strings in the code is called 'magic strings'. The same logic also applies for the numbers: 'magic numbers'

https://dev.to/ruben_alapont/magic-numbers-and-magic-strings-its-time-to-talk-about-it-ci2